src/io/PipeWriter.zig

```diff
@@ -891,6 +891,7 @@ pub fn WindowsBufferedWriter(
 ) type {
     return struct {
         source: ?Source = null,
+        is_fd: ?bool = null,
```

Comment:
this variable name doesn't say what it is used for

it's also an optional bool which takes up extra space

instead, couldn't this be an enum key on `Source`? like `owned_file` to say "a file which should be closed"